integration/openshift/e2e/run_tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+source /etc/os-release || source /usr/lib/os-release
+script_dir="$(realpath $(dirname $0))"
+source "$script_dir/../../../lib/common.bash"
+
+readonly runner="${script_dir}/runner.sh"
+runner_args=""
+
+info "Run OpenShift e2e tests"
+
+if [ "$OPENSHIFT_TESTS_DRY_RUN" == "true" ];then
```

Comment:
Who sets `OPENSHIFT_TEST_DRY_RUN`?

(asking because if some poor user sets it, they may be surprised if setting it to `1` does not work)

Also a bit confused by having both `OPENSHIFT_TEST_DRY_RUN` and `dry_run` in the scripts.

Reply:
Agreed. If we really need both, I'd do something like the following as early in the script as possible:

```bash
[ "$OPENSHIFT_TEST_DRY_RUN" = true ] && dry_run="true" and then only refer to `$dry_run` if possible.
```

Reply:
> Who sets `OPENSHIFT_TEST_DRY_RUN`?

Oh, I forgot to initialize the variable to `false` if not set. The caller of `make openshift-e2e` can export this variable to get the tests running on dry-run mode.

ps: The `CI` variable should be initialized likewise, and I will fix that.

> 
> (asking because if some poor user sets it, they may be surprised if setting it to `1` does not work)
> 
> Also a bit confused by having both `OPENSHIFT_TEST_DRY_RUN` and `dry_run` in the scripts.

Ok, the `runner.sh` is the "backend" script which just run the tests. While developing that script I found handy to have this `dry_run` variable set (which turn on the dry-run mode), because I didn't always needed to run all the tests for some unrelated changes.

I thought that users in general could be interested on this dry-run feature as well. But now that you asked, maybe I should simply remove it...if someone finds value on it then later I can put it back. What do you think @c3d @jodh-intel ?

Reply:
No objection to removing it, no objection to leaving it either.